bin/test

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
 
-python -m unittest
+if [ -z "$1" ]; then
+  FILES="tests/test_*.py"
+else
+  FILES="$1"
+fi
+python -m unittest $FILES
```

Comment:
this makes it so we can run the tests one by one or as a suite and pass `-v` for logging if/when we need it